realm/realm-library/src/objectServer/java/io/realm/mongodb/mongo/MongoCollection.java

```diff
@@ -805,4 +815,106 @@ public ResultT run() {
             }
         });
     }
+
+    /**
+     * Watches a collection. The resulting stream will be notified of all events on this collection
+     * that the active user is authorized to see based on the configured MongoDB Realm rules.
+     *
+     * @return a task that provides access to the stream of change events.
+     */
+    public RealmEventStreamTask<DocumentT> watch() {
+        return new RealmEventStreamTaskImpl<>(getNamespace().getFullName(),
+                new RealmEventStreamTaskImpl.Executor<DocumentT>() {
+                    @NonNull
```

Comment:
See `@Nonnull`-related comment below -  in this case, you just have to delete the annotation, no further action is required.